appveyor.yml

```diff
@@ -15,7 +15,7 @@ environment:
     # See: http://stackoverflow.com/a/13751649/163740
     CMD_IN_ENV: 'cmd /E:ON /V:ON /C .\tests\scripts\appveyor\run_with_env.cmd'
 
-  PYTEST: py.test --maxfail 5 --durations=10 --junitxml=junit-results.xml
+  PYTEST: py.test -n3 --maxfail 5 --durations=10 --junitxml=junit-results.xml
```

Comment:
Be very careful. When I last tested (#2024), xdist caused seemingly random test failures on windows. Unless these problems have been fixed, I wouldn't do this.